arches/app/media/js/models/card-widget.js

```diff
@@ -27,6 +27,16 @@ define([
                 'sortorder': null,
                 'disabled': false
             };
+
+            // Workaround to issue with widgets not being loaded in workflow (https://community.archesproject.org/t/issue-with-workflows-in-7-5/2270)
+            // This is a temporary fix until we can figure out why widgets are not being loaded.
+            if (!document.widgets) {
```

Comment:
This works but is inefficient for merging into core since regardless of if `widgets` exist as a component-level import they'll get re-required here and hung on the document. 

is if possible to include something like:

```
            if (!widgets && !document.widgets) {
                document.widgets = require('widgets');
                widgets = document.widgets;
            }
```

or it could maybe even be simplified to:

```
            if (!widgets) {
                widgets = require('widgets');
            }
```

Reply:
@chrabyrd Refactored as requested.  Checked the workflows and they're still working as expected